src/executor/graph_executor.cc

```diff
@@ -54,6 +54,14 @@ GraphExecutor::~GraphExecutor() {
   }
 }
 
+inline bool SharableStorage(NDArrayStorageType stype) {
```

Comment:
nit: const NDArrayStorageType?

Reply:
it's pass by value. I guess it doesn't matter that much?

Reply:
Not used anymore?